cpu/cc26x0/include/cc26x0_uart.h

```diff
@@ -72,8 +72,8 @@ typedef struct {
 #define UART_FR_RXFF            0x40
 #define UART_FR_TXFE            0x80
 
-#define UART_LCRH_PEN           0x1
-#define UART_LCRH_EPS           0x2
+#define UART_LCRH_PEN           0x2
+#define UART_LCRH_EPS           0x4
 #define UART_LCRH_RXFE          0x4
```

Comment:
Confirmed with the datasheet, Maybe remove the `UART_LCRH_RXFE` since it really part of the `FR` register.

Reply:
Good call, removed.